### PR TITLE
Added use case 7: Secure offline communication for home automation

### PR DESCRIPTION
**Pull request by KillerX**
Hey,

This is my first attempt to contribute to something like this so please excuse beginner mistakes.

The use case deals with home automation systems where a a direct (https) communication between two local devices is wanted or even required due to bad or even non-existent internet connection.

### CODE REVIEW

**Comment by tomoyukilabs** [comment]
@KillerX Thanks so much for your PR. I would like to ask you a question, so could you check the question inline?

**Comment by KillerX** [comment]
@tomoyukilabs I'm most likely missing something but I just see 2 references to "a question inline" but I have not found a question here. Can you please help me find it so I can clarify? Thanks.

**Comment by tomoyukilabs** [comment]
@KillerX Can you check https://github.com/httpslocal/usecases/pull/8#discussion_r131878961 ?

File: UseCases.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ if she usually posts photos to the online service from her smartphone directly,
 
 ![uc06](figs/uc06.jpg)
 
+## UC-07: Secure offline communication for home automation
+
+A user sets up a home automation gateway is under normal circumstances using HTTPS to securely accept commands via a remote server.
```
[the remark this answers is not in the document]
Comment (KillerX) [reply UseCases.md]: 1. Yes, all communication to the remote server is via HTTPS.
2. In this case the most of the devices talking to the gateway are using other protocols, like Z-Wave and ZigBee (in our particular use case). Thus the question does not really apply. BUT as far as general encryption is concerned, we are striving towards having all channels encrypted. The "ethernet" side of the gateway should communicate via HTTPS everywhere, but  currently for APP<->Gateway directly this is not possible in a completely offline solution.

[the remark this answers is not in the document]
Comment (tomoyukilabs) [reply UseCases.md]: @KillerX Thanks a lot for your comment. As you explained, most of devices can speak non-HTTP protocols such as Z-Wave, ZigBee, etc., so a home automation gateway, which translates HTTP and non-HTTP protocols into each other, is necessary for apps using HTTP(S) to control such kind of devices.

My intention of the question above is to clarify which would talk HTTPS in the home network. If I understand your proposal correctly, the app (web and/or native) seems to need to talk to the gateway directly in the home network when the gateway cannot communicate with the remote server due to interruption at gateway's internet connection. Is it right?

[the remark this answers is not in the document]
Comment (KillerX) [reply UseCases.md]: Yes, that is correct. One additional option would also be custom hardware with WiFi (for example and arduino based remote control) that make use of the HTTPS API directly.

**Pull request merged**